includes/templates/single-web-story.php

```diff
@@ -26,7 +28,10 @@
 
 the_post();
 
-?>
-<!DOCTYPE html>
-<?php
-the_content();
+$post = get_post();
+
+if ( $post instanceof WP_Post ) {
+	echo '<!DOCTYPE html>';
+	$renderer = new Story_Renderer( $post );
+	echo $renderer->render();
```

Comment:
Should tabs be used here?

Reply:
Why not? 🤔 

We use spaces for JS, tabs for PHP.

Reply:
Sorry, the GitHub diff view made it look like spaces were used.